foil.tex

```diff
@@ -2,8 +2,10 @@
 % chktex-file 3
 % chktex-file 8
 % chktex-file 10
+% chktex-file 12
 % chktex-file 13
 % chktex-file 17
+% chktex-file 24
```

Comment:
When running linter, it outputs the following warnings:
```
Warning 24 in foil.tex line 82: Delete this space to maintain correct pagereferences.
make: *** [Makefile:134: tex] Error 2
    \label{f:airfoil_shape_image}  
   ^
Warning 24 in foil.tex line 160: Delete this space to maintain correct pagereferences.
    \label{f:cubic_bezier_curve}  
   ^
Warning 12 in foil.tex line 197: Interword spacing (`\ ') should perhaps be used.
where $A \in \mathbb{R}^{m \times n}$ and $b \in \mathbb{R}^m$. when $m>n$, 
```

Therefore, I temporarily suppress the warnings.

Reply:
Good catch.  Could you please create a GitHub issue for @Gene0315 to resolve the formatting issues?  The GitHub issue should include the error messages.